### PR TITLE
Force notifications to "expire" if they're 60+ days old

### PR DESCRIPTION
**Pull request by mjoyce91**

### CODE REVIEW

File: src/actions/notifications.js [hunk 1]
```diff
@@ -103,10 +104,16 @@ export function unsetNotificationsCount() {
...
+  const today$ = new Date();
+  const past60days = subDays(today$, 60).toJSON();
+  return past60days;
+};
+
+export function notificationsCountFetchData(useDateRange = true) {
   return (dispatch) => {
     if (hasValidToken()) {
-      api().get('/notification/?limit=1&is_read=false')
+      api().get(`/notification/?limit=1&is_read=false${useDateRange ? `&date_created__gte=${getDateRange()}` : ''}`)
```
Comment (elizabeth-jimenez) [inline src/actions/notifications.js:116]: `date_created__gte` is this special syntax? I can only find `date_created` on the api side 🤔

Comment (mjoyce91) [reply src/actions/notifications.js:116]: It should appear in swagger. Its part of the rest framework

Comment (elizabeth-jimenez) [reply src/actions/notifications.js:116]: ah i see it! Don't see a Swagger for Notifications for local, on my side, so was looking at the params. I see it used in other ones tho

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🍥 👍

**Comment by elizabeth-jimenez** [comment]
A fun to know (or maybe i just find it fun 👀 ) the subdays() calculates to 4 hours slow. I was trying to see the notification removes itself "live" but it kept not triggering bc of that lol

**Comment by mjoyce91** [comment]
> A fun to know (or maybe i just find it fun 👀 ) the subdays() calculates to 4 hours slow. I was trying to see the notification remove itself "live" but it kept not triggering bc of that lol

4 hours is how far off EST is from GMT so that could be why

**Pull request merged**